### PR TITLE
Fail on non git directories

### PR DESCRIPTION
**Pull request by JPeer264**
These tests assume that your home directory is not a git repo. If your home dir is a git repo your tests might fail.

### CODE REVIEW

**Comment by JPeer264** [comment]
The tests are failing on Node v4. I will fix that tomorrow ✌️

File: test/prompConfig.js [hunk 1]
```diff
@@ -1,11 +1,32 @@
+import fs from 'fs-extra';
 import test from 'ava';
 import path from 'path';
 import chalk from 'chalk';
 import json from 'json-extra';
+import os from 'os';
```
Comment (rudolfsonjunior) [inline test/prompConfig.js]: I would suggest to import `fs-extra` after `chalk` and `os` as the first. to keep the import statement structure more clear.

File: test/prompConfig.js [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
 import getConfig from '../lib/getConfig';
 import { choices, questions } from '../lib/promptConfig';
 
 const cwd = process.cwd();
+const homedir = os.homedir();
+const randomString = Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 10);
```
Comment (rudolfsonjunior) [inline test/prompConfig.js]: why are you using a random string and still append `.back`?

File: test/cli.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import path from 'path';
+import test from 'ava';
```
Comment (rudolfsonjunior) [inline test/cli.js]: To keep the structure more clear I would suggest to import `ava` first.

File: lib/cli.js [hunk 4]
```diff
@@ -24,6 +25,8 @@ updateNotifier({ pkg }).notify();
 
 if (argv.v) {
   console.log(`v${pkg.version}`);
+} else if (!isGit()) {
```
Comment (rudolfsonjunior) [inline lib/cli.js]: Do we use `{}` now for one-line stmt or not? I would suggest that we use the same structure in the whole project.

File: test/prompConfig.js (hunk 2 above)
Comment (JPeer264) [inline test/prompConfig.js]: To see that it is a backup, if something goes wrong

File: lib/cli.js (hunk 4 above)
Comment (JPeer264) [inline lib/cli.js]: It looks a bit weird if the else got multiple lines.

Actually I read something on the Airbnb Style Guide (https://github.com/airbnb/javascript/issues/445).

File: test/prompConfig.js (hunk 2 above)
Comment (rudolfsonjunior) [inline test/prompConfig.js]: the question is more about why do we need a random string, if we also append .back. what is the benefit?

File: lib/cli.js (hunk 4 above)
Comment (rudolfsonjunior) [inline lib/cli.js]: I don't know, but we should keep the same structure in the entire project. I think if the stmt fits in one line, we should omit the braces.

In this issue they are more talking about to omit the braces or did I not get it right?

File: test/prompConfig.js (hunk 2 above)
Comment (rudolfsonjunior) [inline test/prompConfig.js]: If we need it, maybe a more useful and readable way would to be to append a date or datetime to the file. so the string actually has a meaning.

Comment (JPeer264) [inline test/prompConfig.js]: `.back` is ~always a backup file, if something goes wrong we know that it is a backup

Comment (rudolfsonjunior) [inline test/prompConfig.js]: Yeah, I know. but I am not talking about the `.back` more about the random string, read the answers above.

File: lib/cli.js (hunk 4 above)
Comment (JPeer264) [inline lib/cli.js]: But in this case we would have two different conventions in one `if - else` statement

```js
if (statement) oneLiner();
else if (otherStatement) oneLiner();
else {
  notOneLiner();
}
```

The issue above has no real convention, both is valid.

File: test/prompConfig.js (hunk 2 above)
Comment (JPeer264) [inline test/prompConfig.js]: The benefit is that sometimes a `.back` file already exist. So if one exist we would overwrite the existing one. With the random string we ensure that there is no such file and we are good to go ✌️

Comment (JPeer264) [inline test/prompConfig.js]: > maybe a more useful and readable way would to be to append a date or datetime to the files ending. so the string actually gets a meaning

Good point. I didn't saw this - I will change that soon

File: lib/cli.js (hunk 4 above)
Comment (JPeer264) [inline lib/cli.js]: This will be fixed in another PR, as discussed in person.

File: test/prompConfig.js (hunk 2 above)
Comment (JPeer264) [inline test/prompConfig.js]: I added a datetime and 4 random strings, just in case

**Comment by JPeer264** [comment]
@rudolfsonjunior ping
